Automattic/woocommerce-payments
# Added documentation for report API

Fixes #6363 

#### Changes proposed in this Pull Request

This PR adds documentation for reports API endpoints . Date filters for authorizations API endpoint will not be included into this PR. This is because there is an issue with them (sometimes they can return incorrect data) (https://github.com/Automattic/woocommerce-payments/issues/7445).









#### Testing instructions

- After merge, make sure that you can see the docs here: https://automattic.github.io/woocommerce-payments/#introduction
- Test added API endpoints and compare parameters and responses schema, do they match and is something is missing.

-------------------


**Post merge**
